src/devtools/client/inspector/rules/new-rules.js

```diff
@@ -565,14 +560,14 @@ class RulesView {
     }
 
     if (!element) {
-      this.store.dispatch(disableAllPseudoClasses());
-      this.store.dispatch(updateAddRuleEnabled(false));
-      this.store.dispatch(updateClasses([]));
+      // this.store.dispatch(disableAllPseudoClasses());
+      // this.store.dispatch(updateAddRuleEnabled(false));
+      // this.store.dispatch(updateClasses([]));
       this.store.dispatch(updateRules([]));
       return;
     }
 
-    this.pageStyle = element.inspectorFront.pageStyle;
```

Comment:
I think this might have been causing some issues...

Reply:
We don't have this.pageStyle anymore in Replay. This would've represented the StyleRuleActor, which provides methods for editing CSS, which we haven't implemented in Replay. In the existing Rules view, this is also undefined. We will want to clean up this.pageStyle in a followup.